File: skills/astra-toolkit/clients/go/examples/tables-create-vector-index-default.go
package main

import (
	"context"
	"log"
	"os"

	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	ctx := context.Background()

	// Get an existing table
	client := astra.NewClient()

	database := client.Database(
		os.Getenv("ASTRA_DB_API_ENDPOINT"),
		options.API().SetToken(os.Getenv("ASTRA_DB_APPLICATION_TOKEN")),
	)

	table := database.Table("**TABLE_NAME**")

	// Index a vector column
	err := table.CreateVectorIndex(
		ctx,
		"**INDEX_NAME**",
		"**VECTOR_COLUMN_NAME**",
	)
	if err != nil {
		log.Fatal(err)
	}
}

File: skills/astra-toolkit/clients/python/examples/tables-create-vector-index-default.py
import os
from astrapy import DataAPIClient

# Get an existing table
client = DataAPIClient()
database = client.get_database(
    os.environ["ASTRA_DB_API_ENDPOINT"], token=os.environ["ASTRA_DB_APPLICATION_TOKEN"]
)
table = database.get_table("**TABLE_NAME**")

# Index a vector column
table.create_vector_index(
    "**INDEX_NAME**", column="**VECTOR_COLUMN_NAME**"
)
